datadog/resource_datadog_dashboard.go

```diff
@@ -193,7 +198,7 @@ func updateDashboardLists(d *schema.ResourceData, providerConf *ProviderConfigur
 	}
 }
 
-func loadDatadogDashboard(d *schema.ResourceData, dashboard datadogV1.Dashboard) error {
+func loadDatadogDashboard(d *schema.ResourceData, dashboard *datadogV1.Dashboard) error {
```

Comment:
Small nit, can we name this how you're doing the newer ones; i.e. monitor is going to have - `updateMonitorState`